### ISSUE TITLE
can't compile and install with Meson installed from pip

### ISSUE THREAD

**Issue by cobalt2727** [realmazharhussain/gdm-settings#105]
### What happened?

Any time I try running `meson install -C build` it throws me an error claiming the `Build Directory has been generated with Meson version 0.63.2, which is incompatible with the current version 0.61.2.`

### Expected behavior

I'd expect that it would install properly

### To Reproduce

follow the steps to compile gdm-settings on the wiki

### Detailed Explaination (Optional)

```ruby
cobalt@ideapad-flex-5:~/gdm-settings$ meson install -C build
ninja: Entering directory `/home/cobalt/gdm-settings/build'
ninja: no work to do.
Installing subdir /home/cobalt/gdm-settings/src/pages to /usr/local/lib/python3/dist-packages/gdm_settings/pages
Installation failed due to insufficient permissions.
Attempting to use polkit to gain elevated privileges...
ninja: Entering directory `/home/cobalt/gdm-settings/build'
ninja: no work to do.

ERROR: Build directory has been generated with Meson version 0.63.2, which is incompatible with the current version 0.61.2.

cobalt@ideapad-flex-5:~/gdm-settings$ meson --version
0.63.2

cobalt@ideapad-flex-5:~/gdm-settings$ whereis meson
meson: /usr/bin/meson /usr/share/meson /home/cobalt/.local/bin/meson /usr/share/man/man1/meson.1.gz
```

### Application Version

latest git commit

### Source

source code

### Environment

```css
             /////////////                cobalt@ideapad-flex-5 
         /////////////////////            --------------------- 
      ////////767////////////////         OS: Pop!_OS 22.04 LTS x86_64 
    //////7676767676///////////////       
   /////76767//7676767//////////////      
  /////767676////76767///////////////      
 ///////767676///76767.///7676////////     
/////////767676//76767///767676////////    
//////////76767676767////76767/////////    
///////////76767676//////7676//////////   DE: GNOME 42.3.1 
////////////,7676,///////767///////////    
/////////////*7676///////76////////////   
///////////////7676////////////////////   
 ///////////////7676///767////////////    
  //////////////////////'////////////      
   //////.7676767676767676767,//////      
    /////767676767676767676767/////       
      ///////////////////////////         
         /////////////////////
             /////////////
```

**Comment by realmazharhussain** [comment]
As @eli-schwartz said, the problem is that you have two different versions of `meson` installed on your system and one is only available for the user `cobalt`.

In order to solve the issue, uninstall user-only version of meson by running the following command;

```bash
pip3 --user uninstall meson
```

And then regenerate the gdm-settings build directory with the following command (after `cd ~/gdm-settings`)

```bash
meson setup build --wipe
```

**Comment by cobalt2727** [comment]
Is there a way to run the install command that doesn't use meson, out of curiosity?

**Comment by eli-schwartz** [comment]
The install command is generated as a python pickle of internal meson classes, which is iterated over and copied into the destdir, or in some cases, executed as install scripts. It also handles generating the `ninja uninstall` data as a log of installed files etc.

tl;dr no.

What you can do, is install meson as root with sudo pip (don't do this if you have other options or aren't prepared to accept the consequences of clashing with the distro package manager!) or follow the instructions also on that page to create a zipapp or run python from the source tree.

**Comment by cobalt2727** [comment]
Gotcha, thanks for the info!
